packages/zoe/test/unitTests/zoe/test-burnInvitation.js

```diff
@@ -0,0 +1,62 @@
+// @ts-check
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava';
+
+import { makeIssuerKit, AssetKind, AmountMath } from '@agoric/ertp';
+
+import { burnInvitation } from '../../../src/zoeService/offer/burnInvitation';
+
+test('burnInvitation', async t => {
```

Comment:
Would you mind adding a test with multiple invitations?

Reply:
Sure thing

Reply:
Just added a test that tries to burn multiple invitations at once. Let me know if you meant something else!